packages/react-popover/src/components/Popover/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './Popover';
-export * from './Popover.types';
+// Explicit exports to ommit PopoverCommons
+export type { PopoverSize, PopoverProps, PopoverState, OnOpenChangeData, OpenPopoverEvents } from './Popover.types';
```

Comment:
nit: Order these alphabetically
```suggestion
export type { OnOpenChangeData, OpenPopoverEvents, PopoverProps, PopoverSize, PopoverState} from './Popover.types';
```